spec/features/changing_answers_spec.rb

```diff
@@ -199,26 +199,10 @@
       expect(claim.student_loan_plan).to eq StudentLoan::PLAN_1
     end
 
-    scenario "user can change the answer to identity questions that were not acquired from GOV.UK Verify" do
-      claim.update!(govuk_verify_fields: ["first_name", "surname", "address_line_1", "postcode", "date_of_birth"])
-      visit claim_path(StudentLoans.routing_name, "check-your-answers")
-
-      expect(page).to have_content(I18n.t("questions.payroll_gender"))
-      expect(page).to have_selector(:css, "a[href='#{claim_path(StudentLoans.routing_name, "gender")}']")
-
-      find("a[href='#{claim_path(StudentLoans.routing_name, "gender")}']").click
-      choose "I don't know"
-      click_on "Continue"
-
-      expect(claim.reload.payroll_gender).to eq("dont_know")
-    end
-
-    scenario "user can change identity details when GOV.UK Verify was skipped" do
+    scenario "user can change identity details" do
```

Comment:
I'd simply go with a deletion, eg, "user can change the answer to identity questions", so the language is inline with existing specs.

Reply:
ok, good catch